src/StorageCache.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace BlackBox;
```

Comment:
After reflection I think this should go in its own `BlackBox\Cache` namespace. For now the root only contains the interface which makes it very easy to pick up.